cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -139,7 +145,7 @@ func (c *CStorPoolController) cStorPoolEventHandler(operation common.QueueOperat
 		// Check if pool is not imported/created earlier due to any failure or failure in getting lease
 		// try to import/create pool here as part of reconcile.
 		if IsPendingStatus(cStorPoolGot) {
-			status, err := c.cStorPoolAddEventHandler(cStorPoolGot)
+			status, err := c.cStorPoolAddEvent(cStorPoolGot)
```

Comment:
this should be AddEventHandler

Reply:
Sorry sir while resolving conflicts I missed it, Updated it in the latest commit